presentation/src/main/java/com/mashup/gabbangzip/sharedalbum/presentation/ui/common/PicPhotoCard.kt

```diff
@@ -21,35 +28,93 @@ import androidx.compose.ui.unit.dp
 import coil.compose.AsyncImage
 import com.mashup.gabbangzip.sharedalbum.presentation.R
 import com.mashup.gabbangzip.sharedalbum.presentation.theme.Gray0
+import com.mashup.gabbangzip.sharedalbum.presentation.theme.Gray80
+import com.mashup.gabbangzip.sharedalbum.presentation.theme.PicTypography
+import com.mashup.gabbangzip.sharedalbum.presentation.ui.main.grouphome.model.CardBackImage
 import com.mashup.gabbangzip.sharedalbum.presentation.ui.main.grouphome.model.GroupInfo
 import com.mashup.gabbangzip.sharedalbum.presentation.ui.model.GroupKeyword
+import com.mashup.gabbangzip.sharedalbum.presentation.ui.model.GroupStatusType
 import com.mashup.gabbangzip.sharedalbum.presentation.ui.model.PicPhotoFrame
 import com.mashup.gabbangzip.sharedalbum.presentation.utils.StableImage
+import com.mashup.gabbangzip.sharedalbum.presentation.utils.rippleClickable
 
 @Composable
 fun PicPhotoCard(
     modifier: Modifier,
     groupInfo: GroupInfo,
+    content: @Composable BoxScope.() -> Unit,
+    eventName: String = "",
+    onCreateEvent: () -> Unit = {},
 ) {
-    PicPhotoCardFrame(
+    PicPhotoCardContainer(
         modifier = modifier,
         keywordType = groupInfo.keyword,
-        frameResId = groupInfo.frontImageFrame.frameResId,
     ) {
-        AsyncImage(
-            modifier = Modifier.matchParentSize(),
-            model = groupInfo.cardFrontImageUrl,
-            contentScale = ContentScale.Crop,
-            contentDescription = stringResource(R.string.group_main_picture),
-        )
+        if (groupInfo.status == GroupStatusType.NO_PAST_AND_CURRENT_EVENT) {
+            CardTopTitle(
+                modifier = Modifier
+                    .align(Alignment.TopCenter)
+                    .padding(top = 37.dp),
+                text = "이벤트를 만들어보세요",
+            )
+        }
+
+        if (groupInfo.status == GroupStatusType.NO_CURRENT_EVENT) {
+            CardTopTitle(
+                modifier = Modifier
+                    .align(Alignment.TopCenter)
+                    .padding(top = 37.dp),
+                text = groupInfo.recentEventDate,
+            )
+        }
```

Comment:
```suggestion
    val topTitleText = remember(groupInfo.status) {
        when (groupInfo.status) {
            GroupStatusType.NO_PAST_AND_CURRENT_EVENT -> "이벤트를 만들어보세요"
            GroupStatusType.NO_CURRENT_EVENT -> groupInfo.recentEventDate
            else -> ""
        }
    }

    PicPhotoCardContainer(
        modifier = modifier,
        keywordType = groupInfo.keyword,
    ) {
        if (topTitleText.isNotBlank()) {
            CardTopTitle(
                modifier = Modifier
                    .align(Alignment.TopCenter)
                    .padding(top = 37.dp),
                text = topTitleText,
            )
        }
```
CardTopTitle 에 대한 코드가 공통으로 사용되고 있는 것 같은데 약간 이런느낌으로 text 만 조건에 따라 들어가도록 작성해보는 것은 어떨까?

Reply:
아 굳굳 저거 작성하면서도 같이 쓰는 방법이 없을까 고민했었는데 ㅋㄷㅋㄷ 수정할게~ 나이스~